docs/source/models.rst

```diff
@@ -1,44 +1,59 @@
-Models
-======
+Architectures
+=============
+
+Implementation
+------------------
 
 :mod:`ResNet`
----------------------------
+^^^^^^^^^^^^^
 
 .. autoclass:: ivadomed.models.ResNet
 
 
 :mod:`DenseNet`
----------------------------
+^^^^^^^^^^^^^^^
 
 .. autoclass:: ivadomed.models.DenseNet
 
 
 :mod:`Unet`
----------------------------
+^^^^^^^^^^^
 
 .. autoclass:: ivadomed.models.Unet
 
 
 :mod:`FiLMedUnet`
----------------------------
+^^^^^^^^^^^^^^^^^
 
 .. autoclass:: ivadomed.models.FiLMedUnet
 
 
 :mod:`HeMISUnet`
----------------------------
+^^^^^^^^^^^^^^^^
 
 .. autoclass:: ivadomed.models.HeMISUnet
 
 
 :mod:`UNet3D`
----------------------------
+^^^^^^^^^^^^^
 
 .. autoclass:: ivadomed.models.UNet3D
 
 
 :mod:`Countception`
----------------------------
+^^^^^^^^^^^^^^^^^^^
 
 .. autoclass:: ivadomed.models.Countception
 
+Models
```

Comment:
it's a bit "hidden"-- i suggest putting it as main section, and renaming it "Pre-trained models"